src/osd/PG.h

```diff
@@ -557,7 +557,7 @@ class PG : public DoutPrefixProvider {
     return get_pgbackend()->get_is_recoverable_predicate();
   }
 protected:
-  OSDMapRef last_persisted_osdmap_ref;
+  epoch_t last_persisted_osdmap;
```

Comment:
osd/PG: keep epoch, not map ref, of last osdmap for lsat persisted epoch
    
    No need to pin the map in memory!

Reply:
I know the commit says that, but don't we need to refer to all the following maps?
...no, no we don't, because we've already done the processing and just haven't persisted that we've done so. Okay.